src/TLuaInterpreter.cpp

```diff
@@ -3203,92 +3203,212 @@ int TLuaInterpreter::hideUserWindow(lua_State* L)
     return 0;
 }
 
+// No documentation available in wiki - internal function
+void TLuaInterpreter::setBorderSize(lua_State* L, int size, int position, bool updateEvent)
+{
+    Host& host = getHostFromLua(L);
+    // position: 0 = top, 1 = right, 2 = bottom, 3 = left
+    switch(position) {
+        case 0: host.mBorderTopHeight = size; break;
+        case 1: host.mBorderRightWidth = size; break;
+        case 2: host.mBorderBottomHeight = size; break;
+        case 3: host.mBorderLeftWidth = size; break;
+    }
+    if (updateEvent) {
+        int x, y;
+        x = host.mpConsole->width();
+        y = host.mpConsole->height();
+        QSize s = QSize(x, y);
+        QResizeEvent event(s, s);
+        QApplication::sendEvent(host.mpConsole, &event);
+    }
+}
+
+// Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#setBorderSizes
+int TLuaInterpreter::setBorderSizes(lua_State* L)
+{
+    int numberOfArguments = lua_gettop(L);
+    int sizeTop = 0;
+    int sizeRight = 0;
+    int sizeBottom = 0;
+    int sizeLeft = 0;
+    switch(numberOfArguments) {
+        case 0: break;
+        case 1: {
+            if (!lua_isnumber(L, 1)) {
+                lua_pushfstring(L, "setBorderSizes: bad argument #1 value (new size as number expected, got %s!)", luaL_typename(L, 1));
+                return lua_error(L);
+            } 
+            sizeTop = lua_tonumber(L, 1);
+            sizeRight = lua_tonumber(L, 1);
+            sizeBottom = lua_tonumber(L, 1);
+            sizeLeft = lua_tonumber(L, 1);
+            break;
+        }
+        case 2: {
+            if (!lua_isnumber(L, 1)) {
+                lua_pushfstring(L, "setBorderSizes: bad argument #1 value (new heights size as number expected, got %s!)", luaL_typename(L, 1));
```

Comment:
I get where you are coming from, but 'new height as number' would be more intuitive to read and would be consistent with the tense we use in all other messages. I had to think about it the first time I saw it